sectionsUtilityTool/build.sh

```diff
@@ -1,23 +1,19 @@
 #!/bin/sh --
 
 #Helper script for use on build server.
-#
 # Intended to be run after the Maven build is successful.
-#
 
 # Debugging: -x to enable, +x to disable
 set +x
 
 timestamp=$(date +%Y%m%d%H%M%S)
-
-# configuration-files.txt contains list of files to archive.
-# Use "-C <dir>" to change to <dir> before processing remaining files.
-#tar --exclude .svn -cf target/configuration-files.${timestamp}.tar \
-# $(cat configuration-files.txt)
-
+cd sectionsUtilityTool
 cd target
-
 warFilename=$(ls *.war | head -1)
 targetFilename=$(basename ${warFilename} .war)
+#GIT_BRANCH =origin/TLUNIZIN-424 or origin/master jenkins environmental variable to get git branch
+branch=${GIT_BRANCH}
```

Comment:
* As a quick test i echo ${GIT_BRANCH} in the Execute shell window on jenkins configuring just for tags. I get tags/<tagname>. So  the logic to break at '/' is still valid. So it works
* being a newbie to shell programming i opted this route 
  branch=${GIT_BRANCH}
```if [ -n "$branch" ]; then
btemp=$(basename ${branch} /)
else
btemp="local"
fi
mv ${targetFilename}.war ${targetFilename}.${btemp}.${timestamp}.war ```